packages/emails/lib/generateIcsString.ts

```diff
@@ -65,6 +65,10 @@ const generateIcsString = ({
     recurrenceRule = new RRule(event.recurringEvent).toString().replace("RRULE:", "");
   }
 
+  const isOrganizerExempt = process.env.ORGANIZER_EMAIL_EXEMPT_DOMAINS?.split(",").some((domain) =>
+    event.organizer.email.toLowerCase().endsWith(domain.toLowerCase())
+  );
+
```

Comment:
can we move `ORGANIZER_EMAIL_EXEMPT_DOMAINS` to a constants and set default value to empty so that this doesn't throw some error if someone doesn't have `ORGANIZER_EMAIL_EXEMPT_DOMAINS` se